src/udev/udev-builtin-path_id.c

```diff
@@ -635,8 +636,24 @@ static int find_real_nvme_parent(sd_device *dev, sd_device **ret) {
                 return r;
 
         /* If the 'real parent' is (still) virtual, e.g. for nvmf disks, refuse to set ID_PATH. */
-        if (path_startswith(devpath, "/devices/virtual/"))
-                return -ENXIO;
+        if (path_startswith(devpath, "/devices/virtual/")) {
+		const char *syspath;
```

Comment:
`syspath` is set but not used. Please drop this and `sd_device_get_syspath()` below.